### DISCUSSION TITLE
MediaControlWidget

### DISCUSSION THREAD

**Discussion by Heinerbub** [Zefau/ioBroker.jarvis#1133] [Ideas / Feature Requests]
Hallo Zefau, es wäre toll wenn auch weitere Player unterstützt werden.

Ich nutze zB den Logitech Media Server (LMS) /Squeezbox
https://www.npmjs.com/package/iobroker.squeezeboxrpc

**Comment by Zefau** [comment]
https://github.com/Zefau/ioBroker.jarvis/issues/1134
